backend/oasst_backend/scheduled_tasks.py

```diff
@@ -67,6 +69,23 @@ def hf_feature_extraction(text, message_id, api_client):
         logger.error(f"Could not extract embedding for text reply to {message_id=} with {text=} by.error {str(e)}")
 
 
+@shared_task(name="update_search_vectors")
+def update_search_vectors(batch_size: int) -> None:
+    logger.info("update_search_vectors start...")
+    try:
+        with default_session_factory() as session:
+            to_update: list[Message] = session.query(Message).filter(Message.search_vector.is_(None)).all()
```

Comment:
Should we limit the messages returned from db here? On the first run, it will load about 160k messages at once, which might break the server

Reply:
Yes, good point, I intended to put a limit here for the batch size but I guess I didn't